lib/ecto/schema.ex

```diff
@@ -255,6 +255,8 @@ defmodule Ecto.Schema do
   * `__schema__(:primary_key)` - Returns a list of primary key fields (empty if there is none);
 
   * `__schema__(:fields)` - Returns a list of all non-virtual field names;
+  * `__schema__(:db_fields)` - Returns a list of all field names as stored in db;
+  * `__schema__(:aliases)` - Returns a list of fields with their changed source;
```

Comment:
I was referring only to what we use in the reflection. I would avoid `source` exactly because it's already quite an overloaded term and is used primarily to refer to the table name. It might be fine within the limited scope of `field` macro, but I think it's confusing for the reflection mechanism since you have much less context there.